src/pages/product/[itemId].tsx

```diff
@@ -26,6 +32,12 @@ const ProductDetails = ({ itemId }: Props) => {
     }
   }, [dispatch, itemId, item]);
 
+  useEffect(() => {
```

Comment:
If we don't need this, please remove it (since you are using this value in the _app.tsx file.